### PR TITLE
Update lb

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Dum4G** [comment]
It is Al Maaref, is it not?

File: channels/lb.m3u [hunk 1]
```diff
@@ -3,6 +3,8 @@
 https://svs.itworkscdn.net/aghanilive/aghanilive/playlist.m3u8
 #EXTINF:-1 tvg-id="AlIttihad.lb" tvg-name="Al Ittihad" tvg-country="LB" tvg-language="Arabic" tvg-logo="http://alittihad.tv/assets/images/logo.png" group-title="General",Al Ittihad (552p)
 https://live.alittihad.tv/ittihad/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="LB" tvg-language="Arabic" tvg-logo="" group-title="Religious",Almaaref
+http://38.96.175.40:1935/liveTrans/ngrp:channel23_all/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/lb.m3u:7]: I don't seem to have any video from it.

Comment (LaneSh4d0w) [reply channels/lb.m3u:7]: image

Comment (Sphinxroot) [reply channels/lb.m3u:7]: logo if needed https://i.imgur.com/QYCJljN.png

but on site https://almaaref.ch/live/  i get https://5e74a9d684b2e.streamlock.net/liveTrans/ngrp:channel23_all/playlist.m3u8

https://live.almaaref.tv/  seized

**Comment by tilfizyawn** [comment]
Thanks everyone

File: channels/lb.m3u (hunk 1 above)
Comment (tilfizyawn) [reply channels/lb.m3u:7]: It works fine for me
https://streamtest.in/logs/http-38-96-175-40-1935-livetrans-ngrp-channel23_all-playlist-m3u8

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**